packages/deployment/upgrade-test/Makefile

```diff
@@ -74,4 +69,7 @@ run_test:
 run_bash:
 	$(RUN) --entrypoint /bin/bash $(REPOSITORY):$(dockerLabel)
 
+shell:
+	docker exec -it `docker ps --latest --format json | jq -r .Names` bash
```

Comment:
A possible alternative here could be something like:

`docker exec -ti $(docker ps -q -f ancestor=agoric/upgrade-test:agoric-upgrade-11) bash` 

Where you replace `agoric-upgrade-11` with `TARGET` similar to other commands.

Reply:
latest is a problem for me, I'm a heavy docker user and this definitely might not be my latest container

Reply:
I figure `latest` will be what most people want and those who don't (because they use Docker extensively) are sophisticated enough to something else.

Thanks for the simpler way to get the container name, @raphdev . I looked but didn't find it.

I don't think we want to hard-code the ancestor, because it's something to have to remember to change and it's not always the target wanted.

this works,
```
shell:
	docker exec -it `docker ps -q -f ancestor=agoric/upgrade-test:$(TARGET)` bash
```
but then it requires the user to specify the target.

Reply:
we can always iterate on this if/when it becomes a problem